catenets/models/torch/pseudo_outcome_nets.py

```diff
@@ -331,8 +329,8 @@ def predict(self, X: torch.Tensor, return_po: bool = False, training: bool = Fal
             raise NotImplementedError(
                 "PseudoOutcomeLearners have no Potential outcome predictors."
             )
-        if not training:
-            self._te_estimator.model.eval()
+        if not training and hasattr(self._te_estimator, "eval"):
```

Comment:
Why do we have the setting to Eval mode only for SLearner? I think we'd probably need it for all methods? (@kamilest can you comment on this?)